lambda-function/src/api/MeasureServiceApi.ts

```diff
@@ -19,7 +19,10 @@ export class MeasureServiceApi {
       console.log("Measure transferred successfully.");
       return response.data;
     } catch (error: any) {
-      console.log("Failed to transfer the measure over to MADiE: ", JSON.stringify(error.response?.data));
+      console.log(
+        "Failed to transfer the measure (measure namee: " + measure.measureName + ") over to MADiE: ",
```

Comment:
looks like typo in word: name**e**

Reply:
Yeah you are right, sorry about it. Just corrected it. Thanks!